services/pom.xml

```diff
@@ -246,7 +246,10 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-model-storage-private</artifactId>
         </dependency>
-
+        <dependency>
+        	<groupId>org.keycloak</groupId>
+        	<artifactId>keycloak-config-api</artifactId>
+        </dependency>
```

Comment:
This introduces a dependency on a Quarkus dist module in one of the key server modules. IMHO the generic server modules should remain independent of any dist-specific modules. That said, even though `keycloak-config-api` is currently in the Quarkus module, it's not really Quarkus-dependant. It's a generic server config API. Therefore it should be ok to keep as is.

Reply:
@vmuzikar right, this seemed to be the lesser of several evils:
- have services depend upon keycloak-config-api - at least in theory this seems conceptually correct, but as you point out config-api is under quarkus. Moving keycloak-config-api away from quarkus is possible. The dependency on infinispan-commons is unused. However the quarkus-opentelemetry dependency is used for one enum - so that would have to be moved to being defined directly in the PropertyMapper layer. Alternatively we just change the dependency scope for these to something like provided so that it doesn't introduce any transitive considerations.
- have keycloak-config-api depend upon services, this seems odd given how broad services is
- duplicate the default values
- introduce yet another module

Reply:
Works for me as is. If we see issues with that we can change the scope to provided, as you suggested.